src/global/coin-type.ts

```diff
@@ -228,7 +228,7 @@ export class CoinType {
     defaultDecimals : 18
   }, {
     name            : CoinName[CoinName.Dash],
-    addressFormat   : "^X[a-km-zA-HJ-NP-Z1-9]{25,34}$", //TODO
+    addressFormat   : "^X7[a-km-zA-HJ-NP-Z1-9]{25,34}$",
```

Comment:
question. is `X7` a new address format for dash or just something we haven't added?

Reply:
oh, shit. good catch... got it wrong. should be: 

`"^[X7][a-km-zA-HJ-NP-Z1-9]{25,34}$"`

'7' addresses are dash p2sh.

Reply:
ahh ok cool